### PR TITLE
ci: deploy Lambdas via GitHub Actions

### PR DESCRIPTION
**Pull request by smauermann**

### CODE REVIEW

**Comment by Blightwidow** [comment]
@smauermann Ping me when the PR is ready for review

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        zip -qq -r ./bundle.zip ./
+      shell: bash
+    - name: S3 copy
+      run: |
+        cd ${{ inputs.lambda-dir }}
+        aws s3 cp ./bundle.zip s3://${{ inputs.s3-bucket }}/${{ inputs.s3-key }}
+      shell: bash
+    - name: update Lambda
+      run: |
+        aws lambda update-function-code --function-name ${{ inputs.lambda-name }} \
```
Comment (skenmy) [inline action.yml:47]: Does `aws lambda update-function-code` cause the running version to update?

**Review by rodrigodealer — APPROVED** [review]
Looks good!

File: action.yml (hunk 1 above)
Comment (smauermann) [reply action.yml:47]: I assumed that, let me verify.

Comment (smauermann) [reply action.yml:47]: In short, yes. As we are not using Lambda versioning, there is only one version that is running (`$Latest`), which is being updated.

**Comment by smauermann** [comment]
@Blightwidow @skenmy please have another look, cheers

**Review by skenmy — APPROVED** [review]

**Pull request merged**